ground/src/main/res/values/strings.xml

```diff
@@ -144,4 +145,6 @@
     <item quantity="one">%d submission</item>
     <item quantity="other">%d submissions</item>
   </plurals>
+  <string name="loi_name_dialog_title">Name this location</string>
+  <string name="loi_name_dialog_body">A new data collection site will be created.</string>
```

Comment:
If we are going full Jetpack Compose way, we can skip putting our string here, as it requires extra conversion due to the view system and XML.

We can go for `const val` and use the string in our Jetpack Compose UI.

Reply:
Thank you! For now I will keep the string convention because it's fully supported in Compose and it enables us to better handle translations (if we get around to it).

https://developer.android.com/jetpack/compose/resources